<map id="pip._internal.exceptions.DirectoryUrlHashUnsupported" name="pip._internal.exceptions.DirectoryUrlHashUnsupported">
<area shape="rect" id="node2" href="$classpip_1_1__internal_1_1exceptions_1_1HashError.html" title="pip._internal.exceptions.\lHashError" alt="" coords="23,259,185,300"/>
<area shape="rect" id="node3" href="$classpip_1_1__internal_1_1exceptions_1_1InstallationError.html" title="pip._internal.exceptions.\lInstallationError" alt="" coords="23,169,185,211"/>
<area shape="rect" id="node4" href="$classpip_1_1__internal_1_1exceptions_1_1PipError.html" title="pip._internal.exceptions.\lPipError" alt="" coords="23,80,185,121"/>
</map>
